Software/backend/rest_api/Node.js/README.md

```diff
@@ -1,17 +1,122 @@
-<<<<<<< Updated upstream
-<!-- press ctrl+K then V to open a preview of the MarkDown file  -->
-=======
->>>>>>> Stashed changes
-# Page Main Title 
-short description of the directory's subject
+# Node.js and REST API 
+Here you will find information about REST API, Node.js servers and even some useful tool to become a professional coder.
 
-# Table Of Contents
+## Table Of Contents
 
-* [subject](#subject) 
+* [What is API](#What-is-API?) 
 
-* [subject2](#subject2) 
+* [What is REST](#What-is-REST)
 
-# Subject
-* [outer resource](http://...)  
-short description if required
-* [local resource](./resourceName.someFile)
\ No newline at end of file
+* [HTTP Methods - CRUD](#HTTP-Methods---CRUD)
+
+* [Node.js](#Node.js)
+
+* [Useful packages](#Useful-packages)
+
+* [Build your own REST API using node.js and express](#Build-your-own-REST-API-using-node.js-and-express)
+
+* [More information](#More-information)
+
+---
+
+## What is API?
+API or Application Programming Interface is a set of definitions and protocols that developers can use without
+the need to write any code.
+
+https://en.wikipedia.org/wiki/API
+
+https://www.freecodecamp.org/news/what-is-an-api-in-english-please-b880a3214a82/
+
+https://www.youtube.com/watch?v=s7wmiS2mSXY&ab_channel=MuleSoftVideos
+
+https://www.youtube.com/watch?v=tgbRY96q-KM&ab_channel=WebDevSimplified
+
+## What is REST?
+REST or Representational State Transfer is is a software architectural style that defines a set of constraints to be used for creating web services.
+
+https://www.youtube.com/watch?v=rtWH70_MMHM&ab_channel=edureka%21
+
+https://www.youtube.com/watch?v=7YcW25PHnAA&ab_channel=WebConcepts
+
+https://www.youtube.com/watch?v=SLwpqD8n3d0&ab_channel=ProgrammingwithMosh
+
+https://www.youtube.com/watch?v=6sUbt-Qp6Pg&ab_channel=WebDevSimplified
+
+https://www.youtube.com/watch?v=Q-BpqyOT3a8&ab_channel=TraversyMedia
```

Comment:
if all these videos and links are basically the same, only include the best ones.
if they have unique qualities and are worth keeping, say it .
like-
long technical information, short practical explanation,  step by step tutorial.